<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsAdd</class>
 <widget class="QWidget" name="settingsAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название сервера</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название города</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Идентификатор города</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Адрес сервера</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Агент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Куки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Контрольная сумма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Идентификатор пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Идентификатор ассы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Название темы на форуме</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Идентификатор темы форума</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="serverName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="townName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="townId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="serverUrl"/>
       </item>
       <item>
        <widget class="QLineEdit" name="userAgent"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cookie"/>
       </item>
       <item>
        <widget class="QLineEdit" name="checksum"/>
       </item>
       <item>
        <widget class="QLineEdit" name="userId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="corpId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="themeName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="theneId"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Генерировать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
